README.md

```diff
@@ -78,145 +135,201 @@ See [third_party_integration](./third_party_integration) for examples of code mo
-
-[Object detection (3rd-party training pipelines)](#object-detection-mmdetection)
-
-[Instance Segmentation (3rd-party training pipelines)](#instance-segmentation-mmdetection)
-
-
+Results achieved using sample scripts, example patches to third-party repositories and NNCF configuration files provided 
+with this repository. See README.md files for [sample scripts](#model-compression-samples) and [example patches](#third-party-repository-integration) 
+to find instruction and links to exact configuration files and final checkpoints.
+- [PyTorch models](#pytorch-models)
+  * [Classification](#pytorch_classification)
+  * [Object detection](#pytorch_object_detection)
+  * [Semantic segmentation](#pytorch_semantic_segmentation)
+  * [Natural language processing (3rd-party training pipelines)](#pytorch_nlp)
+  * [Object detection (3rd-party training pipelines)](#pytorch_object_detection_mmdetection)
+  * [Instance Segmentation (3rd-party training pipelines)](#pytorch_instance_segmentation_mmdetection)
+- [TensorFlow models](#tensorflow-models)
+  * [Classification](#tensorflow_classification)
+  * [Object detection](#tensorflow_object_detection)
+  * [Instance segmentation](#tensorflow_instance_segmentation)  
+
+### PyTorch models
+
+<a name="pytorch_classification"></a>
 #### Classification
 
-|Model|Compression algorithm|Dataset|PyTorch FP32 baseline|PyTorch compressed accuracy|
-| :---: | :---: | :---: | :---: | :---: |
-|ResNet-50|INT8|ImageNet|76.16|76.42|
-|ResNet-50|INT8 (per-tensor only)|ImageNet|76.16|76.37|
-|ResNet-50|Mixed, 44.8% INT8 / 55.2% INT4|ImageNet|76.16|76.2|
-|ResNet-50|INT8 + Sparsity 61% (RB)|ImageNet|76.16|75.43|
-|ResNet-50|INT8 + Sparsity 50% (RB)|ImageNet|76.16|75.55|
-|ResNet-50|Filter pruning, 40%, geometric median criterion|ImageNet|76.16|75.62|
-|Inception V3|INT8|ImageNet|77.34|78.25|
-|Inception V3|INT8 + Sparsity 61% (RB)|ImageNet|77.34|77.58|
-|MobileNet V2|INT8|ImageNet|71.93|71.35|
-|MobileNet V2|INT8 (per-tensor only)|ImageNet|71.93|71.3|
-|MobileNet V2|Mixed, 46.6% INT8 / 53.4% INT4|ImageNet|71.93|70.92|
-|MobileNet V2|INT8 + Sparsity 52% (RB)|ImageNet|71.93|71.11|
-|SqueezeNet V1.1|INT8|ImageNet|58.24|58.28|
-|SqueezeNet V1.1|INT8 (per-tensor only)|ImageNet|58.24|58.26|
-|SqueezeNet V1.1|Mixed, 54.7% INT8 / 45.3% INT4|ImageNet|58.24|58.9|
-|ResNet-18|XNOR (weights), scale/threshold (activations)|ImageNet|69.8|61.63|
-|ResNet-18|DoReFa (weights), scale/threshold (activations)|ImageNet|69.8|61.61|
-|ResNet-18|Filter pruning, 40%, magnitude criterion|ImageNet|69.8|69.26|
-|ResNet-18|Filter pruning, 40%, geometric median criterion|ImageNet|69.8|69.32|
-|ResNet-34|Filter pruning, 40%, geometric median criterion|ImageNet|73.3|72.73|
-|GoogLeNet|Filter pruning, 40%, geometric median criterion|ImageNet|69.75|68.82|
-
+|PyTorch Model|<img width="115" height="1">Compression algorithm<img width="115" height="1">|Dataset|Accuracy (Drop) %|
+| :---: | :---: | :---: | :---: |
+|ResNet-50|INT8|ImageNet|76.42 (-0.26)|
```

Comment:
Perhaps we should specify the drop in a separate column? Anyway, could you please update the `tools/update_eval_results.py` so that it generates tables consistent with any of the two new formats?

Reply:
@AlexKoff88, will we use `Accuracy (Drop) %` format for the main Readme?

Reply:
I support, this is very informative.